.github/workflows/reuse.yaml

```diff
@@ -102,7 +102,7 @@ jobs:
         id: ko-build-dev
         if: ${{inputs.tag == 'image' && steps.extract_branch.outputs.branch == 'main'}}
         run: |
-          echo "::set-output name=digest::$(REGISTRY=ghcr.io/${{github.repository}} GITHUB_TOKEN=${{github.token}} make ${{inputs.publish_command}}-dev)"
+          echo "::set-output name=digest::$(REGISTRY=ghcr.io GITHUB_TOKEN=${{github.token}} make ${{inputs.publish_command}}-dev)"
```

Comment:
REGISTRY is already ghcr.io by default, and you're setting `KO_DOCKER_REPO` in the ko build steps correctly now, so this could just be removed right?

In any case, this looks like a good change to me, thanks for tracking that down.